composer.json

```diff
@@ -5,33 +5,17 @@
   "type": "library",
   "authors": [
     {
-      "name": "Tim Lytle",
```

Comment:
let's keep a list of all people that worked on this - maybe replace their Vonage email address with their Twitter link

Reply:
In line with the other libraries, I was thinking we could add them to the contributors list? I'll need to change the others to keep a continuous list I think